File: Python/A_NivelIniciante/Exercicio_1043.py
# -*- coding: cp1252 -*-
# -*- encoding: utf-8 -*-
#url do problema - https://www.urionlinejudge.com.br/judge/pt/problems/view/1043
linha1 = raw_input()
A,B,C = linha1.split(" ")
A = float(A)
B = float(B)
C = float(C)
def testador(lado):
    if (B - C)<lado<B + C:
    	return True
    return False
if testador(A)==True:
    print("Perimetro = %.1f" %(A+B+C))
else:
    print("Area = %.1f" %((A+B)/2 * C))
    

File: Javascript/A_NivelIniciante/Exercicio_1043.js
var input = require("./readFile.js").input;
var lines = input.split('\n');
var lines = lines[0].split(" ")
var a = lines[0]*1
var b = lines[1]*1
var c = lines[2]*1
if((a+b) > c && (a+c) > b && (c+b) > a){
    console.log("Perimetro = " + (a+b+c).toFixed(1))
}else{
    console.log("Area = " + ((a+b)/2 * c).toFixed(1))
}